user-guide/modules/ROOT/pages/boost-history.adoc

```diff
@@ -25,6 +25,34 @@ One of the defining characteristics of Boost is its strict peer review process.
 
 Boost has had a profound impact on the pass:[C++] language and its community. Many of the libraries and concepts introduced by Boost have been adopted into the pass:[C++] Standard Library, including smart pointers, regular expressions, and the boost:lambda[] function syntax. Boost has also been a fertile ground for experimenting with new ideas and techniques, which have later made their way into the pass:[C++] language itself, such as boost:type_traits[] and boost:mpl[].
 
+=== Developer Conference
+
+_BoostCon_, initiated in 2007, was a conference dedicated to the discussion and improvement of both Boost libraries and pass:[C++] software development in general. In 2012, BoostCon was rebranded as _pass:[C++] Now_ to reflect a broader scope. While Boost libraries remained a crucial part of the discussion, the conference expanded to cover popular and advanced pass:[C++] techniques.
```

Comment:
Are there links we can include here?

Reply:
@alandefreitas  added links to Boostcon 2012, cppnow